lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
I think `applyDefaultIgnorePatterns` would be a better name for this option. The current name doesn't specify what patterns are being applied, and the meaning isn't obvious from context if a user doesn't specify `ignorePattern`.